bin/ldnode.js

```diff
@@ -97,6 +97,11 @@ var argv = require('nomnom')
     full: 'default-app',
     help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
   })
+  .option('signup', {
```

Comment:
Makes sense. (So this is more for https://github.com/solid/solid/issues/7). 
Can we rename it to `--create-admin` then?